engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/videoSettings/VideoSettingsScreen.java

```diff
@@ -64,26 +65,56 @@ public VideoSettingsScreen() {
     @Override
     @SuppressWarnings("unchecked")
     public void initialise() {
+
         UIDropdown<Preset> videoQuality = find("graphicsPreset", UIDropdown.class);
         if (videoQuality != null) {
+            videoQuality.setOptionRenderer(new StringTextRenderer<Preset>() {
```

Comment:
refactor to `TranslateableStringTextRenderer<T extends TranslateableItem>` or something like that